TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -785,11 +785,12 @@ internal void AcknowledgeCopyChanges(List<ChangedCopy> changedCopies)
                     // Evict from target project cache only
                     evictions[workspaceProject.Project.CopyCache].Add(changedCopy.CopyName);
                 }
-                else
+                else if (!WorkspaceProjectStore.IsKnownProject(changedCopy.OwnerProject))
                 {
-                    // Inconsistent notification from client, the target project could not be found
-                    LoggingSystem.LogMessage(LogLevel.Warning, $"Copy to WorkspaceProject mismatch: could not find project '{changedCopy.OwnerProject}'.");
+                    // Inconsistent notification from client, the target project is unknown
+                    LoggingSystem.LogMessage(LogLevel.Error, $"Copy to WorkspaceProject mismatch: project '{changedCopy.OwnerProject}' is unknown.");
```

Comment:
Maybe we could have a fixed error message and the variable part (with project key) in the context data
It will be easier then to filter on this message when analyzing logs.

Reply:
Yes ! Done.